web/themes/custom/cloudy/dist/bootstrap.bundle.js

```diff
@@ -23,5 +23,5 @@
;o.enumerable=o.enumerable||!1,o.configurable=!0,"value"in o&&(o.writable=!0),Object.defineProperty(e,o.key,o)}}var sn=".".concat("bs.toast"),ln=i.a.fn.toast,cn="click.dismiss".concat(sn),un="hide".concat(sn),fn="hidden".concat(sn),hn="show".concat(sn),dn="shown".concat(sn),pn={animation:"boolean",autohide:"boolean",delay:"number"},mn={animation:!0,autohide:!0,delay:500},gn=function(){function e(t,n){!function(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}(this,e),this._element=t,this._config=this._getConfig(n),this._timeout=null,this._setListeners()}var t,n,o;return t=e,o=[{key:"VERSION",get:function(){return"4.6.0"}},{key:"DefaultType",get:function(){return pn}},{key:"Default",get:function(){return mn}},{key:"_jQueryInterface",value:function(t){return this.each((function(){var n=i()(this),o=n.data("bs.toast"),r="object"===tn(t)&&t;if(o||(o=new e(this,r),n.data("bs.toast",o)),"string"==typeof t){if(void 0===o[t])throw new TypeError('No method named "'.concat(t,'"'));o[t](this)}}))}}],(n=[{key:"show",value:function(){var e=this,t=i.a.Event(hn);if(i()(this._element).trigger(t),!t.isDefaultPrevented()){this._clearTimeout(),this._config.animation&&this._element.classList.add("fade");var n=function(){e._element.classList.remove("showing"),e._element.classList.add("show"),i()(e._element).trigger(dn),e._config.autohide&&(e._timeout=setTimeout((function(){e.hide()}),e._config.delay))};if(this._element.classList.remove("hide"),s.reflow(this._element),this._element.classList.add("showing"),this._config.animation){var o=s.getTransitionDurationFromElement(this._element);i()(this._element).one(s.TRANSITION_END,n).emulateTransitionEnd(o)}else n()}}},{key:"hide",value:function(){if(this._element.classList.contains("show")){var e=i.a.Event(un);i()(this._element).trigger(e),e.isDefaultPrevented()||this._close()}}},{key:"dispose",value:function(){this._clearTimeout(),this._element.classList.contains("show")&&this._element.classList.remove("show"),i()(this._element).off(cn),i.a.removeData(this._element,"bs.toast"),this._element=null,this._config=null}},{key:"_getConfig",value:function(e){return e=on(on(on({},mn),i()(this._element).data()),"object"===tn(e)&&e?e:{}),s.typeCheckConfig("toast",e,this.constructor.DefaultType),e}},{key:"_setListeners",value:function(){var e=this;i()(this._element).on(cn,'[data-dismiss="toast"]',(function(){return e.hide()}))}},{key:"_close",value:function(){var e=this,t=function(){e._element.classList.add("hide"),i()(e._element).trigger(fn)};if(this._element.classList.remove("show"),this._config.animation){var n=s.getTransitionDurationFromElement(this._element);i()(this._element).one(s.TRANSITION_END,t).emulateTransitionEnd(n)}else t()}},{key:"_clearTimeout",value:function(){clearTimeout(this._timeout),this._timeout=null}}])&&an(t.prototype,n),o&&an(t,o),e}();i.a.fn.toast=gn._jQueryInterface,i.a.fn.toast.Constructor=gn,i.a.fn.toast.noConflict=function(){return i.a.fn.toast=ln,gn._jQueryInterface}}]);
```

Comment:
It doesn't seem like the js and css assets should have changed in this PR. 🤔

Reply:
Yep, I thought the same thing. I reverted the file changes several times, but they always same back every time I ran `lando refresh`. I even just doublechecked on master with a fresh DB and the files get changed with `lando refresh` so it seems that they are indeed out of date or there's something else messed up in the project.

Reply:
Huh. Sure enough. If I run the commands on my local against `master` I get the same change. It's almost as if it is backing out the change that came along with the recent yarn upgrade, but a yarn audit doesn't seem to be a problem. Weird.